### PR TITLE
SDPA-456 Takes package error from log

### PR DESCRIPTION
**Pull request by naomiaro**
Check if package type is a takes package.

### CODE REVIEW

File: apps/packages/takes_package_service.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get_take_package_id(self, item):
         :return: _id of the package or None
         """
         takes_package = [package.get(PACKAGE) for package in item.get(LINKED_IN_PACKAGES, [])
-                         if package.get(PACKAGE_TYPE)]
+                         if package.get(PACKAGE_TYPE) is TAKES_PACKAGE]
```
Comment (mdhaman) [inline apps/packages/takes_package_service.py]: With the above change, text items are getting delivered to `Digital Subscribers`. Hence you had to change the number in the test. 

The change should be `if package.get(PACKAGE_TYPE) == TAKES_PACKAGE`. I want to see what the data looks like. Can you export the document from mongo?

Comment (naomiaro) [inline apps/packages/takes_package_service.py]: woops was foiled by my equality test in console.

http://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce

I don't have the data, I can try to get it from PA if it's still in their dev instance. 

These were in the logs

[32m11:05:00 work.1 | [0mHTTP Exception 403 has been raised: Multiple takes found for item: urn:newsml:localhost:2016-05-18T12:52:30.494683:af7d8887-8e97-47e3-b1cd-b5555fd67ea8 level=ERROR process=Worker-2
[32m11:05:00 work.1 | [0m403: Multiple takes found for item: urn:newsml:localhost:2016-05-18T12:52:30.494683:af7d8887-8e97-47e3-b1cd-b5555fd67ea8 level=ERROR process=Worker-2

**Comment by petrjasek** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Pull request merged**